configuration/wireless/access-point-bridged.md

```diff
@@ -174,6 +174,38 @@ To use the 5 GHz band, you can change the operations mode from `hw_mode=g` to `h
 
 Note that when changing the `hw_mode`, you may need to also change the `channel` - see [Wikipedia](https://en.wikipedia.org/wiki/List_of_WLAN_channels) for a list of allowed combinations.
 
+Check iptables are not filtering bridge traffic with br_netfilter
+
+
+```
+cat /proc/sys/net/bridge/bridge-nf-call-arptables 
+1
+cat /proc/sys/net/bridge/bridge-nf-call-iptables
+1
+cat /proc/sys/net/bridge/bridge-nf-call-ip6tables
+1
+```
+if they return 1 or true as above you can temporarily disable for testing by these commands:
+```
+sudo echo 0 | tee /proc/sys/net/bridge/bridge-nf-call-arptables
+0
+sudo echo 0 | tee /proc/sys/net/bridge/bridge-nf-call-iptables
+0
+sudo echo 0 | tee /proc/sys/net/bridge/bridge-nf-call-ip6tables
+0
+```
+or edit 
```

Comment:
Editing this file will make the change permanent, but you've said above that these commands are only temporary.  I think you should probably make it clear that changing `sysctl.conf` should only be done once the reader is happy to make the change persist across a reboot.